packages/block-library/src/gallery/edit.js

```diff
@@ -237,7 +237,11 @@ class GalleryEdit extends Component {
 	componentDidMount() {
 		const { attributes, mediaUpload } = this.props;
 		const { images } = attributes;
-		if ( Platform.OS === 'web' && every( images, ( { url } ) => isBlobURL( url ) ) ) {
+		if (
+			Platform.OS === 'web' &&
+			images && images.length > 0 &&
```

Comment:
Nitpick, but: If we spread a condition across multiple lines, each part should be on its own line:

```diff
- images && images.length > 0 &&
+ images &&
+ images.length > 0
```

There's meant to be guidance around this in the JavaScript coding standards. They appear to be out of date on the live site.

Archive: http://web.archive.org/web/20190412084124/https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/